app/controllers/admins/admins_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Admins
+  class AdminsController < ApplicationController
+    before_action :admin
+
+    def edit_password; end
+
+    def update_password
+      if admin.update_with_password(pass_params)
+        redirect_to admins_users_path
+      else
+        render :edit_password
+      end
+    end
+
+    private
+
+    def pass_params
+      params.require(:admin)
+            .permit(:current_password, :password, :password_confirmation)
+    end
+
+    def admin
+      @admin ||= Admin.find_by(id: params.fetch(:admin_id))
```

Comment:
we should use `current_admin` here, not this `find`

Reply:
can you tell me what happens if `admin_id` is absent?

Reply:
Can not use `current_admin`, because authentication hasn`t implemented yet

Reply:
there will be an error if `admin_id` is absent